### ISSUE TITLE
Help on PixelWeightedFusionSoftmax

### ISSUE THREAD

**Issue by rajuptvs** [coperception/coperception#14]
Hello, 

I was going through the  DiscoGraph paper and the code for the Disconet, Can you please help me understand if the **PixelWeightedFusionSoftmax** is the same as the collaboration graph process for the student model, mentioned in the paper?

Thanking you in advance

**Comment by quan-dao** [comment]
Hi @rajuptvs ,
I think `PixelWeightedFusionSoftmax` and the collaboration graph are the same.

Let F_{ i } denote the BEV feature map of the connected vehicle **i** and F_{ j -> i } is the BEV feature map of the vehicle **j** transformed to the coordinate of vehicle **i** using the pose of vehicle **j** in the frame of vehicle **i**. 

For a pixel **n**, the collaboration graph determines the weight w_{ j } of F_{ j -> i }[ n ] which is used in the weighted sum w_{ j }* F_{ j -> i }[ n ] (for j = 1 to num connected vehicles) for determining the value of pixel **n** in the fused F_{ i }. 

The logits of  w_{ j } (i.e. unnormalized  w_{ j }) is calculated by applying a shared MLP on the stack of  F_{ j -> i }[ n ] and  F_{ i }[ n ]. Because we have many connected vehicles (j >= 1), the logits of w_j are converted to probability by softmax.

**Comment by dekunma** [comment]
Closing this issue due to inactivity. Feel free to reopen it if there are any further questions related to this topic.
